src/commands/aem/rde/history.js

```diff
@@ -41,12 +44,21 @@ class HistoryCommand extends BaseCommand {
       } else if (isNaN(args.id) || parseInt(args.id, 10) < 0) {
         throw new validationCodes.INVALID_UPDATE_ID({ messageValues: args.id });
       } else {
+        result.items = [];
         await this.withCloudSdk((cloudSdkAPI) =>
-          rdeUtils.loadUpdateHistory(cloudSdkAPI, args.id, cli, (done, text) =>
-            done ? this.spinnerStop() : this.spinnerStart(text)
+          rdeUtils.loadUpdateHistory(
+            cloudSdkAPI,
+            args.id,
+            this,
+            (done, text) =>
+              done ? this.spinnerStop() : this.spinnerStart(text),
+            result.items
           )
         );
       }
+      if (flags.json) {
+        return result;
+      }
```

Comment:
Can't we always return the json object? I would assume that oclif ignores it when we're generating text output,  @right?

Reply:
I can evaluate, my changes were simply as non invasive as possible.

Reply:
seems like oclif handles this as expected, so we can always return it.